lighthouse-core/gather/connections/cri.js

```diff
@@ -19,11 +19,23 @@
 const Connection = require('./connection.js');
 const WebSocket = require('ws');
 const http = require('http');
+const log = require('../../lib/log.js');
+
 const hostname = 'localhost';
-const port = process.env.PORT || 9222;
+const CONNECT_TIMEOUT = 10000;
 
 class CriConnection extends Connection {
   /**
+   * @param {number=} port Optional port number. Defaults to 9222;
+   * @constructor
+   */
+  constructor(port) {
```

Comment:
should this take a connectOpts obj instead of named args? seems like our other connections would not take `port` as first arg.